### PR TITLE
Add support for TPU training.

### PR DESCRIPTION
**Pull request by pavithrasv**
Add support for TPU training using cloud TPUs via AI platform training service.

### CODE REVIEW

File: tensorflow_cloud/deploy.py [hunk 1]
```diff
@@ -23,6 +23,14 @@
 from googleapiclient import errors
 
 from . import gcp
+from . import machine_config
+
+try:
+    from tensorflow.python.framework.versions import VERSION
+except ImportError:
+    # Use TF runtime version 2.1 (latest supported) as the default.
```
Comment (fchollet) [inline tensorflow_cloud/deploy.py:31]: 2.2 not supported?

File: tensorflow_cloud/gcp.py [hunk 2]
```diff
@@ -38,13 +38,20 @@ def validate_machine_configuration(
     Args:
         cpu_cores: Number of virtual CPU cores.
         memory: Amount of memory in GB.
-        accelerator_type: 'MachineConfig.AcceleratorType'
+        accelerator_type: 'MachineConfig.AcceleratorType'.
```
Comment (fchollet) [inline tensorflow_cloud/gcp.py:41]: "Instance of ..."?

File: tests/integration/call_run_on_script_with_keras_fit.py [hunk 3]
```diff
@@ -34,16 +34,24 @@
 #     stream_logs=True)
 
 # Automated MirroredStrategy: chief config with multiple GPUs
+# tfc.run(
```
Comment (fchollet) [inline tests/integration/call_run_on_script_with_keras_fit.py:37]: Why the disabled test?

File: tensorflow_cloud/deploy.py (hunk 1 above)
Comment (pavithrasv) [reply tensorflow_cloud/deploy.py:31]: Yes, 2.2 is not supported yet for cloud TPU training using AI platform. It is coming soon.

File: tests/integration/call_run_on_script_with_keras_fit.py (hunk 3 above)
Comment (pavithrasv) [reply tests/integration/call_run_on_script_with_keras_fit.py:37]: We can call tfc.run only once from a python script. Here i am showcasing the various distribution strategies using the same python script. So i have commented out all but one strategy.

**Comment by pavithrasv** [comment]
Addressed comments . Thank you Francois!

**Pull request merged**